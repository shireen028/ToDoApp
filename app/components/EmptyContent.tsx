import React from 'react';
import {Image, StyleSheet, View, Dimensions} from 'react-native';
const {height} = Dimensions.get('window');

export const EmptyContent = () => {
  return (
    <View style={styles.container}>
      <Image style={styles.image} source={require('assets/empty.png')} />
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center',
    justifyContent: 'center',
    backgroundColor: 'white',
    height: height - 56,
  },
  image: {
    width: 300,
    height: 280,
  },
});
