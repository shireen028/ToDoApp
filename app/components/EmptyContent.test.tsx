import 'react-native';
import React from 'react';
import {EmptyContent} from './EmptyContent';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
    renderer.create(<EmptyContent />);
});
