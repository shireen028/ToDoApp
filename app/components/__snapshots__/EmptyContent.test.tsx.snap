// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "height": 1278,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../app/assets/empty.png",
      }
    }
    style={
      Object {
        "height": 280,
        "width": 300,
      }
    }
  />
</View>
`;
